### PR TITLE
add TimeInMilliseconds method

### PR DESCRIPTION
**Pull request by talwai**

### CODE REVIEW

File: statsd/statsd.go [hunk 1]
```diff
@@ -185,6 +185,13 @@ func (c *Client) Set(name string, value string, tags []string, rate float64) err
 	return c.send(name, stat, tags, rate)
 }
 
+// Sends timing information in milliseconds. Flushed by statsd with percentiles, mean and other info
```
Comment (jmoiron) [inline statsd/statsd.go]: golint will complain about this because Go documentation for functions/methods start off like `<FunctionName> ...`

Comment (talwai) [inline statsd/statsd.go]: Fixed

**Comment by jmoiron** [comment]
lgtm :shipit:

**Comment by remh** [comment]
it's interesting to notice that it's the same exact thing as a histogram.

we actually treat |h and |ms the same exact way in the server 
https://github.com/DataDog/dd-agent/blob/master/aggregator.py#L739-L740

I know it's the one type used by etsy statsd but do we really need this ?

**Comment by talwai** [comment]
Admitting that `|ms` is essentially an alias of `|h`, my inclination is to keep this change so that sinks modeled after the [go-metrics StatsdSink](https://github.com/armon/go-metrics/blob/master/statsd.go#L56-L59) can use this client to send an identical set of metric types. I think maintaining parity in the packets themselves is important so as not to mess with any dowstream tooling

**Pull request merged**